### DISCUSSION TITLE
Possibility to associate a "Name" (Payee) on  two "Assett accounts" type - v. 1.6.5. requested

### DISCUSSION THREAD

**Discussion by MAPA2023** [moneymanagerex/moneymanagerex#5912] [Ideas]
Follow the second part "Withdrawal and Deposit" Form

**Comment by tactilis** [comment]
@MAPA2023 
> The idea is to be able to add when needed (therefore the data should not be mandatory) the reference to the Master Data. This would allow you to track a Payee also for transfer operations.

> The purpose of all this is to be able to have Transitional Accounts such as for example preparing an open balance with a customer or supplier and then resetting it.

> Currently in the posting lines the transfer account is highlighted in the "Payee" column but in fact it would not be.

Transfers in MMEX are only between accounts.

If you want a 'Transitional Account' for the purpose you describe, you could simply create an account using the name of the payee in the account name. Then transfer the money to it as required, and when appropriate, transfer it out to the real payee. When you are finished with the account you could mark it as Closed (and optionally hide it from view).

If I understand your need correctly, then maybe the idea of Pots/Piggy Banks/Spaces as requested here https://github.com/moneymanagerex/moneymanagerex/issues/3904 and here https://github.com/moneymanagerex/moneymanagerex/issues/5714 may be what you are looking for.

**Comment by MAPA2023** [comment]
Hi Tactilis and Stein, 

I'll try to explain the request better. In fact, it would be necessary to set up an asset account with the possibility of being seen in the Deposits and Expenses configuration (for example, by marking it in the settings with a flag). The reason is to be able to associate a "Name" (customer or supplier) so as to move it also on two balance sheet accounts. Let me give you a concrete example: at the beginning of the year I have a balance sheet called "Sales 2022" (in the picture below Suppliers). On this account I can handle the opening of a certain name "Names" with a registration. Subsequently during the year I can reset the account with a series of operations, for example from a bank but related to the same name. So subsequently by making an extract that contemplates the "Name" I will see all the movements.

![Transaction 2](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/c45a67f9-0ac7-4b92-8968-331bb9303c2e)


example in other program

![EX other](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/28729ce2-1641-4098-9966-570cb7c8e33c)

Ex. Settings in Asset Account.

![Transaction 3](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/d82e9084-43b1-4a53-9910-e7046c7bff05)

**Comment by tactilis** [comment]
@MAPA2023

To quote from the Documentation:

> "Money Manager Ex (MMEX) is a _**personal**_ money management system..."

It feels as though the need you are describing in https://github.com/moneymanagerex/moneymanagerex/discussions/5912#discussioncomment-5917669 is for a more business orientated accounting/bookkeeping application.

**Reply by MAPA2023** [reply]
Hi Tactilis, I think it's an optimization on this product too which I find extremely interesting otherwise. I use various accounting programs. This request comes from my experience and I find it sensible and complete also for Money Manager. Of course it is not vital but very useful and I think even feasible with little effort.

**Comment by MAPA2023** [comment]
I write an example to describe this need. I want to have a "Client Card" (asset account) on which to transit income and expenses, but to be able to monitor this transfer also related to the "Client Name" (Beneficiary).

So for example I would enter (Deposit) an amount from the "Bank" account to the "Customer Card" account (two asset accounts) but taking into consideration the "Customer" (Beneficiary) itself.
In the "Category" I would therefore go to put the "Customer Card" account as a counterpart, supplying the amount that arrived via the Bank but, and this is the important difference, taking into consideration the "customer's name" (Beneficiary name). The operation is done with a Deposit or on the contrary with a Withdrawal (not a Transfer).

At present I can do all this with two operations, one in Deposit and one in Widrowal using a transitional "Category" (in fact it always remains with a zero balance).

It would be nice to be able to do it all with just one registration. As in the photos for example by marking only the accounts that I want to manage in the Categories popup.

**Reply by tactilis** [reply]
Someone may chose to implement this feature but I believe that the number of people who would have a need to use it is very small.

You have mentioned Clients, Customers, Suppliers, and Sales 2022, so it appears that you are wishing to use MMEX for business purposes . There's nothing wrong with that if it does what you need - but again I would re-iterate that:

> _"Money Manager Ex (MMEX) is a **personal** money management system..."_

Unlike some other accounting applications, MMEX's focus is not on the needs of businesses.

**Comment by MAPA2023** [comment]
Hi Tactilis,

thank you first of all.

I see this change only as an optimization of the flexibility of the product which I personally find very perforated and simple. The introduction of this function makes it possible to associate a "Name" between two Asset accounts type.

The most appropriate way seems to be to be able to include this type of account in the "Category" so as to be able to use it in the procedure of Deposits and Withdrawals.

I hope someone does it by understanding its importance and it would be very convenient for me personally.

Thanks a lot for your availability!

**Comment by MAPA2023** [comment]
Possibility to associate a "Name" (Payee) to 2 "Assett accounts" type

I submit this idea that allows you to get even more flexibility in this beautiful project.

As you know in mmex, an accounting entry of "Deposit" or "Withdrawal" is always based on various factors among which there are mandatory: "Account" (Asset - bank etc..), Name (Payee or From), "Category " (Revenue or Cost).

Well, for example, I need to use an "Asset Account" where I want to convey the money received, for example, from a bank associated with a specific name (From) and then have the costs exit from that account. This allows me to monitor the account for that particular name!

Currently this is not possible with a single transaction that moves "two accounts" of the patrimonial type associated with a "Name" (From or Payee).

Two obvious optimization implementations are possible at the discretion of the engineers:
1a in the "Transfer" procedure, add the "Name" (Payee or From) like to "Category" but it is optional and not mandatory.

2s in the "Deposit" or "Withdrawal" procedure, add in the "Category" the possibility of viewing the balance sheet accounts, for example set to be viewed.

I am attaching hypothetical vision photos.

What do you think?

EX. OPTION A
![Transerf A](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/e055b90b-1bdf-403b-af76-6a3659c16c14)


EX. OPTION B
![Transaction 2](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/0193a6ac-96fc-44a3-ba3e-01dc7575ab35)

**Comment by MAPA2023** [comment]
I did a series of tests and I can confirm that the correct solution is the second hypothesis that I show in the picture below. This solution gives a way to increase, for example, the cash account and to debit, for example, the loan account to the child on which I will then pass all deposits and withdrawals.

![Transaction 2](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/afd2b2a5-1f9a-4efa-be89-c19beb2f68af)

**Comment by n-stein** [comment]
I'll admit not being an accountant by trade I don't fully understand option B. I feel using an account as a category would be confusing to the average user. Also trickier to implement given that account and category IDs overlap.

An optional payee on a transfer seems more straightforward both functionally and technically. It's simply filling an existing data field on the transaction in the manner for which it is designed, not trying to repurpose a field to hold multiple types of data.

**Comment by MAPA2023** [comment]
Hi Stein,

your observation is correct !

For a non-professional user, viewing is simpler this way and still obtains an excellent result, i.e. being able to monitor a certain "Name" (Payee) even for transfers from tow "Asset account" (Cash to Bank ex.)

I therefore confirm that your analysis is correct!

Many thank for your precious support.

Max.

![Transerf A](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/81ed983f-4eaa-40f5-b3e7-d7733e56b79b)

**Comment by tactilis** [comment]
@n-stein
> I'll admit not being an accountant by trade I don't fully understand option B.

I do not understand it either.

@MAPA2023 clearly wants MMEX to do something that it currently does not but has proposed 'solutions' that attempt to shoehorn the new capability into MMEX's existing functionality. This makes for a confusing UI.

It would be better if the need was clearly explained in simple terms:
- What is the real world problem that needs solving?
- Give some simple examples of _what_ information would need to be entered, tracked, and reported on (NB. _what_ not _how_)?
- Explain who would use this capability and when?
- Explain why it is important that the capability be included in a _Personal_ (i.e. not Business) Money Management application?

From a clearly stated need, an optimal UI solution can be evolved.

<br>

@n-stein
> An optional payee on a transfer seems more straightforward both functionally and technically. It's simply filling an existing data field on the transaction in the manner for which it is designed, not trying to repurpose a field to hold multiple types of data.

Maybe...
_Provided that_:
- the approach suggested solves the real world problem;
- does not cause confusion for the thousands of MMEX users who have no need for this capability (since it has never been ask for before);
- and doesn't have unintended effects on existing Reports.

The reason I'm suggesting that we be process driven about this capability is that it affects a very prominent part of the UI that _everyone_ uses and, once a change like this has been added, it is difficult to back it out if causes confusion or does not properly satisfy the need.

**Comment by MAPA2023** [comment]
Hi everyone,

I try to answer the various questions with my poor English and I apologize for that.

What is the problem?
In fact it is not a problem but an important optimization. At the moment it is not possible to associate a "Name" (Payee) in the "Transfer" procedure and therefore on the use of two patrimonial type accounts, which is instead possible in all the other operations, making the latter performing in the analysis in the filters and in all statistical extracts.

What must be entered?
Simply the "Name" (Payee) in the "Transfer" procedure.

Who would use it?
In fact All in my opinion when implemented.

EXAMPLE.
My son Diego gives me € 100.00 to be managed for his expenses and I keep track of his accounting. I then proceed with a simple "Transfer" operation, let them enter my "Cash" account and debit them to the "Family Expenses Account" (Asset in fact an advance or loan account). Soon after, my wife does too. So I have € 200.00 in the Cashier and the same amount in the "Family Account".

Now here's the important part for me! The difference, introducing the change is that on these two "Transfer" operations I was able to monitor "who gave me the money" as is the case on all other operations and therefore be able to analyze and filter them in the various extracts.

Now my example, in some ways even extreme, can currently be resolved with two operations using a transitory income statement (Category). But if the modification on the "Transfer" procedure can be implemented in a simple way, it would be a very good added value.

I personally find that the product that you have created and are bringing aviants with so much effort is very efficient in this respect: easy, flexible and fast to use. Precisely for this reason it is certainly also used by small entrepreneurs who perhaps issue an invoice with an Excel sheet but record the data on this system. I find that my observation can be of help and optimization regardless of ensuring further performance even for that type of user.

I sincerely hope that my observation is intended to provide a starting point for the growth of mmex and above all I hope it can be implemented in a simple way.

Thanks again to all the work team for their dedication and patience with me.

**Reply by tactilis** [reply]
Thank you @MAPA2023

Your example explains your need more clearly.

A question... In your example, is your "Family Expenses Account" a different _real-world_ bank account that you actually transfer the money from your son and wife to? Or is it a 'virtual account', i.e. is it just a means for your to keep track of an amount of money that is being kept for some purpose but which actually still lives in another account?

I can think of different ways of achieving what you want but:
1. The actual method will depend on a full understanding of how you are managing the money; hence the question above.
2. We'd want to find a method that was as flexible as possible so it could handle your use case (tracking payees) as well as related use cases that other people might have (but maybe not involving the names of payees)

**Comment by MAPA2023** [comment]
...
yes, I confirm that it is the SECOND hypothesis: defining a "Count Asset" to be monitored for expenses or revenues based on multiple names but tracing the designated name in the "Transfer" transaction. Now, after a considerable use of mmex in recent months, I think that the solution of inserting the "Payee" in the transaction (for example by forcing the standard) in fact completes the information making the data usable on the various reports.

**Reply by tactilis** [reply]
> _yes, I confirm that it is the SECOND hypothesis_

By which you mean my _"Or is it a 'virtual account', i.e. is it just a means for your to keep track of an amount of money that is being kept for some purpose but which actually still lives in another account?"_

If so, then as I said previously at https://github.com/moneymanagerex/moneymanagerex/discussions/5912#discussioncomment-5916277:

> If I understand your need correctly, then maybe the idea of Pots/Piggy Banks/Spaces as requested here https://github.com/moneymanagerex/moneymanagerex/issues/3904 and here https://github.com/moneymanagerex/moneymanagerex/issues/5714 may be what you are looking for.

It seems to me that you have been proposing a 'solution' that is limited to your particular needs of having a payee name attached to what you regard as a transfer to an 'asset'.

I think we could satisfy a broader range of use cases by having the ability to _allocate_ sums of money to 'Savings Pots', as they are often called in banking apps, even though the money actually still resides in the original real world account, and therefore in the original MMEX account.

I have used the word _allocate_ because it's not really a _transfer_. Transfers are movements of money between MMEX accounts which reflect the movement of money in real-world accounts.

If Savings Pots, or whatever they will be called, are implemented, then you'd be able to create transactions to allocate money to them, optionally with a payee name, or any other identifying string (not limited to payees).

Is this a correct understanding (and generalisation) of your need?

**Comment by MAPA2023** [comment]
I help you understand my requirement, but I think ahe of others, with this example

SLIDE A - I receive a GAS bill (GAs Supplier) but I don't pay it immediately, however I want to monitor its value as well as that of electricity until I pay it. So also the electricity bill etc. In fact, at the end I have a bill that tells me how much more I have to pay. So far it's fine.
![Slide A](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/12fb508c-1e73-4ac4-8e50-41a441b3be3f)

SLIDE B - I register the bill and send the cumulative ENERGY BILLS account to passive, so I will do the same with the other bills, each one in the name of their own Payee.
![Slide B](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/6816b6b0-9af5-470d-815a-5faa9737203e)

SLIDE C - Unfortunately the day has come to pay the gas bill and therefore I pay it. How do I create a transfer (but unfortunately without the Payee). So the result on the "Energy Bill" account coincides but....
![Slide C](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/a86e96cb-d228-42d2-962a-26da13785e02)

SLIDE D - At this point I want to monitor the "Energy bill" account with an extract related to a specific "Payee" ...
![Slide D](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/08388f48-cec5-4f53-baf3-c9adfdcffe35)

SLIDE E - And that's the problem. The extract is missing a piece of data because the transfer does not contemplate it or the "Payee"
![Side E](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/b9f206eb-dd49-486e-8c99-683ff4e4f917)

Now, today I solved this thing with two operations, one in Withdrawal and one in Deposit but in fact it is certain that if you add the Payee in the Transfer operations the added value is there and it is concrete. From everything I've seen, nothing should be accessed on the rest of the reports except an optimization.

Now Tactilis asks you how to monitor the "Energy Bill" account statement for a certain "Payee". I'm not asking a big deal in accounting and we're talking about money movement here as well. What do you think?

Another thing is that from a development point of view it is complex to implement then it's a whole other story...

Thanks as always for your patience

**Comment by tactilis** [comment]
@MAPA2023
> SLIDE A - I receive a GAS bill (GAs Supplier) but I don't pay it immediately, however I want to monitor its value as well as that of electricity until I pay it. So also the electricity bill etc. In fact, at the end I have a bill that tells me how much more I have to pay.

In this case, why don't you simply create the Withdrawal transactions in the account from which you will pay the bills with a future date and leave the transaction Unreconciled?

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/07ec14c6-8b5d-4e72-b837-d52e54e74792)

Or if you want the transactions to stand out, then mark them for Follow Up.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/fa941986-f6b4-45d2-b37c-3277708682c4)

Make sure that `View | Ignore Future Transactions` is not ticked.

Your Transfer of money between accounts seems overly complex, gives rise to the problems you describe, and does not allow MMEX to reflect what is actually happening in your real-world bank account.

**Comment by MAPA2023** [comment]
Thanks for the reply.

Yes, yours is a possibility, but it doesn't determine the cost or revenue based on the date of the document. He bases it on the payment that sooner or later will be done justly. This may or may not be fine. In the example you gave, I ask you: what is the total balance that Gasprom has opened and that I still have to pay (imagine 3 invoices)?

However, now I ask to you what is the reason from an analysis point of view why you didn't plan to put the "Payee" in a Transer when it is useful instead to have it? I'm just asking to understand.

I see only benefits in adding this data.
What are the negative aspects of putting it instead?
I'm not talking about technical negativity on those I don't have expertise, I'm only talking about those relating to the possibility of analyzing data in all its forms...

In summary, why does your answer seem like a no on the request?

**Reply by tactilis** [reply]
@MAPA2023 
> Yes, yours is a possibility, but it doesn't determine the cost or revenue based on the date of the document. He bases it on the payment that sooner or later will be done justly. This may or may not be fine. In the example you gave, I ask you: what is the total balance that Gasprom has opened and that I still have to pay (imagine 3 invoices)?

That's easy.

Consider these 3 future Gasprom invoices:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/6b46c70b-6a0f-4255-9b7e-0ac670501a6d)

You can filter these as shown and MMEX will calculate the Filtered View Balance (i.e. the total of the Gasprom invoices) for you:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/27d10e4f-916c-452f-8c6d-8525b94fb6ac)

Or, instead of filtering, you could manually select the future Gasprom invoice transactions and MMEX will display the total:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/f2c20394-fd87-4550-8cc3-063575cca4a1)

**Reply by tactilis** [reply]
@MAPA2023
> However, now I ask to you what is the reason from an analysis point of view why you didn't plan to put the "Payee" in a Transer when it is useful instead to have it? I'm just asking to understand.

What I have described above is a much simpler solution, using MMEX's existing capabilities, than your 'Transfer' with Payee proposal.

When you transfer between MMEX accounts you are modelling what is happening in your real-world accounts. _You_ are making the transfer between your accounts. There is no payee.

It just does not make sense to me to have a payee.

**Reply by tactilis** [reply]
@MAPA2023
> I see only benefits in adding this data.
What are the negative aspects of putting it instead?

For a start:

- Additional UI complexity for Edit Transaction, Recurring Transactions, Transaction Filter
- Changes required to the Transaction List to be able to show To account _and_ Payee
- Analysing any issues relating to Reports where to the fact that Transfer transactions now have payees
- User confusion as to why, when transferring money between their accounts, a payee field is now displayed
- Additional development effort/testing/future maintenance/documentation (not that we are good at doing the latter)

There are certain to be more.

It feels as though you are wanting to modify MMEX in a way that breaks the simple conceptual model:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/d21c4056-b6e1-457d-b059-3e7dbfdb091a)

In MMEX, the people that give us money or receive our money are regarded as the Payees of the system.
Deposits and Withdrawals have Payees. Transfers are between accounts and don't involve Payees.

**Reply by DevAnyKR** [reply]
Q.
Is there a country where multiple owners are allowed for one account?
In Korea, one account has only one recipient. Even corporate joint management accounts.
So I don't understand the idea of writing a payee separately in a transfer transaction.
It's a hard problem.

**Reply by MAPA2023** [reply]
Hi Martin,

hope it's a good day for you.

Thank you very much for your time and explanations especially the technical one. I agree with the accounting part, but up to a certain point. My opinion remains the same. In fact, all the part of the balances you show is available on screen, but if you launch the printouts and extracts, they don't have this data. Not being able to trace the name in a transaction on two balance sheet accounts from my point of view is a limitation of the system. For example, the fact that to make a transfer you are obliged to put an income statement (Category) on which to pass the operation is apparently an artifice. But this also has a positive side, namely the ability to trace internal transfer movements. On this logic it could (indeed it must where and when necessary) be able to monitor the "Payee". Every accounting has its management aspects as well you know and for my need, but I think also that of many others, this feature could be much appreciated. From what I've seen and the way printable reports are structured, the idea of having this feature gave and added value to mmex.

Now leaving out my personal opinion, the real problem is a technical one which seems to be expensive and delicate. I can't say anything about that. If adding this data creates an inconvenience and instability on the application, then it shouldn't be done.

Today, as already mentioned, I can obtain exactly the result I described to you with three entries, two of which are placed on an economic transitory account (Category) which as such always remains with a zero balance, just as it needs to make mmex for a transfer from one balance sheet account to another. Like I said it was just an optimization.

Thank you very much for the time you have dedicated to me.

I wish the whole team a good job and if I have any other ideas I will share them with you precisely because I strongly believe in the project which is everyone's project.

Best and thanks again.

Max.

**Reply by tactilis** [reply]
@MAPA2023
> to make a transfer you are obliged to put an income statement (Category) on which to pass the operation is apparently an artifice

The approach used by many is to select a category of 'Transfer'. That way it is very clear in the Transaction List that this is a transfer as shown below.

In addition, if you set the Option `Default Transfer Category` to `Last Used`, then every time you create a new transfer, the 'Transfer' category will be automatically filled in for you, so there is no additional effort.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/7aa1871b-8f83-44f5-903c-782d50351728)

<br>

If you wish to further categorise the purpose of your transfers, there is nothing to prevent you adding sub-categories to the Transfer category. For example:

```
- Transfer
-     Car Savings Pot
-     Energy bills Savings Pot
-     Vacation Savings Pot
-     etc
```

**Reply by MAPA2023** [reply]
All clear Martin.
But if the purpose of my transfer is a "Payee" I can't do this. We'll see in the future if anyone else asks and why and if you decide to do it. Thanks again for everything.

**Reply by MAPA2023** [reply]
I'll give you one last example "Devel... Any..." but obvious to all who will read it. I dedicate some time just to let you understand the optimization I'm requesting from the Development Team and because I really like mmex and I want to see it grow.

Let's say we have a series of Clients (Payee) who give me money to get my services. The peculiarity is that the services will be future and therefore not definable when the money is given to me. So I just want to register the entry of money, for example in "Bank" (Asset Account) on an account that I will trivially call "Advances Account" (also Asset account). A passive value is rightly generated on this last account as my potential Payees are in credit to me for the money they have given me but not yet used and available for future services.

The moment comes when I perform my service and therefore generate the real Revenue (or, on the contrary, the cost). The operation will therefore be to go to the "Advance Account" and make a "Deposit" operation using the related "Payee" and the appropriate "Category" based on the type of revenue (ex. Tax Consultancy). By doing so on the "Advance Account" (asset account) I make the revenues pass only when I have actually generated them against money which, however, was given to me previously. This operation correctly reduces the relative balance. Not only on that account do I also see the total value generated by all my "Payees" obviously if the Payee can put them in the first phase of the operation!

Now the question is: how do I do the first part WITH A SINGLE REGISTRATION or how to handle two Asset type accounts (Bank to Advance Account) in relation to a respective "Payee"?
Now it's not possible! Hence my request.

At present, fortunately, I solve the problem with two registrations made in this way:
A - Deposit (on the bank), with the Respective Payee;
B - Withdrawal on the "Advance Account with the Respective Payee.
Note: both operations are performed with the same "Category" (which we will call, for example, "Transitory advance account" as a concept similar to that used by mmex for "Transfer" operations.

Gooo... now I can monitor my Payee with their respective balance in all the printouts and reports that are out there using the powerful filters currently already available. Now I can rightfully "Monitoring" my Payees even if associated with only two balance-type accounts. Now the data is complete!
Surely I don't do alchemy as an example open an asset type account for each Payee by "naming" it with the respective name.

As you can see, luckily it's not a problem. The theme was to resolve everything with a single transaction that takes into account the two balance sheets in play by associating the respective Payee. In fact in line with any other accounting program which, like all of them, has the sole basic purpose of monitoring where the money comes from and what it is spent on.

By everyone and good job.

**Reply by MAPA2023** [reply]
I'm glad you use the same method too (at the moment it's the only way to get this result).
In fact, another example is to register an energy bill for example on 12/31 and pay it in January. The accrual is in December or the current year but the payment is for the following year. Wanting to do it....
 
So the transcriptions would be TWO in this case:
A - "INVOICES account" (asset account), related "Payee", and the "Category" cost of home light.
B - Payment of the invoice with a TRANSFER operation or exit from "Bank Account" to " INVOICES account" with the reference of the "Payee". Analyzing the INVOICES account for a specific Payee, I would have all the movements and relative balance.

It seems simple but Tactilis says there are too many checks to do or technically it would seem like a problem. I'd love to hear @n-stein's opinion on this as well. We will see...
 
Too bad because from my point of view with a simple extra information (the Payee in fact) that we already have and it is also important we would greatly optimize the efficiency of mmex.

We'll see, I hope someone else will also ask, such as a small business owner or a studio that provides services.

By Max

**Comment by MAPA2023** [comment]
@n-stein What do you think about?

**Reply by MAPA2023** [reply]
I'll leave this last note for when you make the decision to implement the request. I did a number of comparisons with various similar systems. In fact, this form comes out as a winner.  The user who decides to manage this procedure certainly knows why it was done. It is therefore not necessary to touch the "Transfer" procedure!

Example with the need to have a Deposit account on services that went to support or receive from different "Payee".

![Transaction 2](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/56ca5338-0111-4ed6-9128-aa7c2114285e)

**Reply by MAPA2023** [reply]
@n-stein What do you thing?

**Comment by n-stein** [comment]
I'm currently working on #5439, perhaps this can solve your problem. You could tag each transaction with the payee name and filter for that tag in the transaction report.

**Reply by MAPA2023** [reply]
@n-stein, thank you very much for warning me. I take it seriously. I looked again is a great modification very useful this too. Good great job !
From what I see it seems to be what in accounting jargon is defined as a "Cost Center" which is certainly an added value, I imagine there will be statistical reports that will include it.

However I don't think it solves my problem which is based on using the "Payee" instead. When you have time please take check my requirement carefully. Choose which may be the most correct path. Seeing other programs is the last above that I have left.
In the example I use the "Deposit" which obviously is a "Withdrawal" in the other account but it is where the operation that must be traced starts from, therefore from my point of view the selectable account type of the obviously exposed "Categories" should only be added separately.
Thanks as always for everything. I have already left a donation but tell me if you want a contribution for this change, I will give it with great satisfaction.

**Comment by MAPA2023** [comment]
@n-stein
I'm following the new TAGS feature very carefully.
Well done you are doing a great job !!!
In fact in relation to my requirement it could fit. In fact I could play Tags like Payee.
However, I ask you when you will have time to consider my request which regardless, from an accounting point of view it is certainly an added value on the program. As already mentioned, the last of my hypotheses is the correct one, i.e. make available in the Category selection the "Equity Asset" accounts that the user decides must be visible there for the management of some particular deposit and withdrawal transactions which, however, take as refer to that type of account. One of the most frequent is the "Advance Account". All my considerations come from my experience and from other products examined.

**Reply by nmsleal** [reply]
@MAPA2023, have you considered using something more suited to do accounting activities, with things like the one you're asking for in order to, I'm supposing, implement double accounting? MMEX is a great personal finance app, flexible enough to do some business-like tasks, but if you start to add this kind of complexity it won't be personal, nor business oriented, anymore...

GNUCash would, probably, be more suitable.

**Reply by MAPA2023** [reply]
Dear nmsleal, 

Thanks for the reply. I have seen and reviewed more than 12 personal and corporate accounting programs and I think it comes out as a winner for a number of reasons mmex. It's not a problem if you don't want to do it. I solved it with two operations, one for deposit and one for withdrawal with a category defined as transitory. As explained, the purpose, even in personal finance, is to move a Payee to an ACCOUNTS account. That's all ! Thanks anyway for the support.

**Comment by MAPA2023** [comment]
Hi @n-stein any new on my request? For example in 2024...
